### PR TITLE
Add Some Entity Equipment Update Packet Mappings

### PR DESCRIPTION
**Pull request by TheBrokenRail**

### CODE REVIEW

File: mappings/net/minecraft/network/packet/s2c/play/EntityEquipmentUpdateS2CPacket.mapping [hunk 1]
```diff
@@ -1,3 +1,8 @@
 CLASS net/minecraft/class_2744 net/minecraft/network/packet/s2c/play/EntityEquipmentUpdateS2CPacket
 	FIELD field_12565 id I
+	FIELD field_25721 equipment Ljava/util/List;
```
Comment (liach) [inline mappings/net/minecraft/network/packet/s2c/play/EntityEquipmentUpdateS2CPacket.mapping]:
```suggestion
	FIELD field_25721 equipments Ljava/util/List;
```

File: mappings/net/minecraft/network/packet/s2c/play/EntityEquipmentUpdateS2CPacket.mapping [hunk 2]
```diff
@@ -1,3 +1,8 @@
...
+	METHOD <init> (ILjava/util/List;)V
+		ARG 1 id
+		ARG 2 equipment
```
Comment (liach) [inline mappings/net/minecraft/network/packet/s2c/play/EntityEquipmentUpdateS2CPacket.mapping]:
```suggestion
		ARG 2 equipments
```

File: mappings/net/minecraft/network/packet/s2c/play/EntityEquipmentUpdateS2CPacket.mapping [hunk 3]
```diff
@@ -1,3 +1,8 @@
...
 	METHOD method_11820 getId ()I
+	METHOD method_30145 getEquipment ()Ljava/util/List;
```
Comment (liach) [inline mappings/net/minecraft/network/packet/s2c/play/EntityEquipmentUpdateS2CPacket.mapping]:
```suggestion
	METHOD method_30145 getEquipments ()Ljava/util/List;
```

File: mappings/net/minecraft/network/packet/s2c/play/EntityEquipmentUpdateS2CPacket.mapping (hunk 1 above)
Comment (TheBrokenRail) [reply mappings/net/minecraft/network/packet/s2c/play/EntityEquipmentUpdateS2CPacket.mapping]: I am pretty sure ``equipment`` is both plural and singular, as in ``I have some equipment in the car.``.

Comment (liach) [reply mappings/net/minecraft/network/packet/s2c/play/EntityEquipmentUpdateS2CPacket.mapping]: Hmm, I think we need a way to distinguish that this is a list than a single equipment.

Comment (TheBrokenRail) [reply mappings/net/minecraft/network/packet/s2c/play/EntityEquipmentUpdateS2CPacket.mapping]: Maybe ``equipmentList``, or ``equipmentSlotToItem`` (bit verbose)?

Comment (TheBrokenRail) [reply mappings/net/minecraft/network/packet/s2c/play/EntityEquipmentUpdateS2CPacket.mapping]: Or ``equipmentUpdates``?

Comment (liach) [reply mappings/net/minecraft/network/packet/s2c/play/EntityEquipmentUpdateS2CPacket.mapping]: `equipmentList` I suppose

Comment (TheBrokenRail) [reply mappings/net/minecraft/network/packet/s2c/play/EntityEquipmentUpdateS2CPacket.mapping]: Done!

**Review by liach — APPROVED** [review]
:heart: Thanks so much!

**Review by YanisBft — APPROVED** [review]

**Pull request merged**